libsql-hrana/src/proto.rs

```diff
@@ -22,6 +22,9 @@ pub struct PipelineRespBody {
     pub results: Vec<StreamResult>,
 }
 
+#[derive(Serialize, Deserialize, prost::Message)]
+pub struct Stats {}
```

Comment:
Where is this used?

Reply:
I believe this was extra, removing it. :)